api.planx.uk/saveAndReturn/validateSession.ts

```diff
@@ -29,12 +30,14 @@ const validateSession = async (
     if (sessionData) {
       // if a user has paid, skip reconciliation steps and return *without* calling updateLowcalSessionData
       if (sessionData.data.govUkPayment?.state?.status === "created") {
-        return res.status(200).json({
+        const responseData = {
           message: "Payment process initiated, skipping reconciliation",
           alteredNodes: null,
           removedBreadcrumbs: null,
           reconciledSessionData: sessionData.data,
-        });
+        };
+        await createAuditEntry(sessionId, responseData, true);
```

Comment:
nit/thought: Certainly not required for this PR, but it might be nice / helpful to have an `AuditService` or similar creating BOPS / Uniform / Reconciliation records going forward.

Reply:
nit: Could we use an options object avoid a boolean param for `reconciliationSkipped` here? Looking at  `createAuditEntry(sessionId, responseData, true)` it's not inherently obvious what the final parameter means.

Alternatively, we could capture this variable and use it for control flow in this function.

```js
const isReconciliationSkipped = someLogic()
if (isReconciliationSkipped) {
  doStuff()
else {
  doOtherStuff()
}
await createAuditEntry(sessionId, responseData, isReconciliationSkipped);

Reply:
rethinking this a bit: since this boolean isn't a super straightforward interpretation here anyways, I've edited the table schema again and made the third argument `message` - which will have the same benefit of letting us filter for analytics but capture more scenarios more clearly, and more easily scale to future reconciliation scenarios. also makes for for much less ambiguous final parameter!

Reply:
Great solution - much more scalable that adding multiple bool columns in future 👍